Missing include on Fedora 27

The include path for QWebEngineView is not configured correctly for Fedora 27. The error is:

```
In file included from qt/moc_spectregui.cpp:9:0:
qt/spectregui.h:9:10: fatal error: QWebEngineView: No such file or directory
 #include <QWebEngineView>
```
The missing files are in

- /usr/include/qt5/QtWebEngineWidgets/qwebengineview.h
- /usr/include/qt5/QtWebEngineWidgets/QWebEngineView

so the correct include would be `QtWebEngineWidgets/QWebEngineView`, or alternatively `/usr/include/qt5/QtWebEngineWidgets` must be included on the compiler level separately. The same problem probably also applies to other libraries like QtGui, QtWidgets, QtWebChannel, QtNetwork.

--- a/configure.ac
+++ b/configure.ac
@@ -88,7 +88,7 @@ if test "x$enable_gui" = "xyes" ; then
     AC_SUBST([LRELEASE])
     AC_MSG_NOTICE([Using Qt5 at $with_qt5])
   ], [
-    PKG_CHECK_MODULES(QT5, [Qt5Core Qt5Gui Qt5Widgets Qt5Network], [AC_MSG_NOTICE([Found Qt5 using pkg-config.])], [
+    PKG_CHECK_MODULES(QT5, [Qt5Core Qt5Gui Qt5Widgets Qt5Network Qt5WebEngineWidgets Qt5WebChannel], [AC_MSG_NOTICE([Found Qt5 using pkg-config.])], [
       PKG_CHECK_MODULES(QT5, [QtCore >= 5 QtGui >= 5 QtWidgets >= 5 QtNetwork >= 5 QtWebEngineWidgets >= 5 QtWebChannel >= 5], [AC_MSG_NOTICE([Found Qt >= 5 using pkg-config.])], [
         AC_MSG_ERROR([Qt5 not found.])
       ])

